docs/source/reference/interoperability.rst

```diff
@@ -66,6 +66,34 @@ In addition, :func:`cupy.asarray` supports zero-copy conversion from Numba CUDA
     x_numba = numba.cuda.to_device(x)  # type: numba.cuda.cudadrv.devicearray.DeviceNDArray
     x_cupy = cupy.asarray(x_numba)  # type: cupy.ndarray
 
+mpi4py
+------
+
+`MPI for Python (mpi4py) <https://mpi4py.readthedocs.io/en/latest/>`_ is a Python wrapper for the Message Passing Interface (MPI) libraries.
+
+MPI is the *de facto* standard for high-performance inter-process communications. Recently several MPI vendors, including Open MPI and MVAPICH, have extended their support beyond the v3.1 standard to enable "CUDA-awareness"; that is, passing CUDA device pointers directly to MPI calls to avoid explicit data movement between the host and the device.
```

Comment:
I agree that it is the de facto standard but probably I would rewrite it as "MPI is the most widely known and used standard for ..."

Reply:
@emcastillo, "most widely used" should be sufficient. The change is made. Please proceed, thanks.